### PR TITLE
updated tile text

### PR DESCRIPTION
**Pull request by david-cabelus-oracle**
I updated the text in the tiles, and I updated the one-line description of Verrazzano.

### CODE REVIEW

**Comment by gilbode** [comment]
@hyder - can you weigh in ?  I know you had some opinions on this

**Comment by hyder** [comment]
Is there a preview of how it would look like?

File: content/en/_index.html [hunk 1]
```diff
@@ -20,41 +20,74 @@ <h1>Verrazzano Enterprise Container Platform</h1>
...
 
 {{< blocks/section color="success" >}}
 
-{{% blocks/tile icon="fab fa-github" title="Contributions welcome!" url="https://github.com/verrazzano/verrazzano" %}}
-We do a [Pull Request](https://github.com/verrazzano/verrazzano/pulls) contributions workflow on **GitHub**. New users are always welcome!
+{{% blocks/tile icon="fa-cogs" title="Day 2 Multi-Cloud Operations" %}}
+<ul align="left">
+    <li>Centralized cross-cloud cluster and application management</li>
+    <li>Centralized automated observability</li>
+    <li>Optimized multicloud application lifecycle</li>
```
Comment (rosemarymarano) [inline content/en/_index.html:69]: -> life cycle (n), lifecycle (adj)

File: content/en/_index.html [hunk 2]
```diff
@@ -20,41 +20,74 @@ <h1>Verrazzano Enterprise Container Platform</h1>
...
+    <li>Use Coherence for in-memory data processing, application performance improvements, and state management</li>
+</ul>
 {{% /blocks/tile %}}
 
 
-{{% blocks/tile icon="fab fa-twitter" title="Follow us on Twitter!" url="https://twitter.com/verrazzano_io" %}}
-For announcement of latest features etc.
+{{% blocks/tile icon="fas fa-chart-line" title="New Application Development" %}}
+<ul align="left">
+    <li>Application modeling for enhanced microservice cross-cloud lifecycle and application portability</li>
```
Comment (rosemarymarano) [inline content/en/_index.html:86]: -> -> life cycle (two words)

**Review by rosemarymarano — APPROVED** [review]
Minor edit, otherwise fine.

**Comment by hyder** [comment]
how do you test this locally to see how it's rendered? is this hugo?

**Comment by MelwynPaul** [comment]
> Centralized automated observabilit

**Comment by MelwynPaul** [comment]
@hyder 
Apologies, I was trying to reply and closed the PR by mistake. 

Yes, you can test this locally. 
Run a local build and go to the main page by selecting the 'VERRAZZANO' icon on the left-corner of the page. 
or, go to http://localhost:1313

**Comment by hyder** [comment]
The wording in the 3 tiles are not consistent. The 1st one is imperative form, the 2nd one is in descriptive, the 3rd one in note. 

Can I suggest the following:

1. The wording should follow a similar style, preferably short imperative
2. Rename "integrated monitoring" to "inbuilt observability"
3. Rename "security" to "security integration"
4. Describe each tile's content succinctly
5. The one-liner description should stand out. It's kinda lost in the blazing light colors.

**Comment by rosemarymarano** [comment]
Yes, Hugo.

File: content/en/_index.html (hunk 1 above)
Comment (david-cabelus-oracle) [reply content/en/_index.html]: Thanks, Rosemary! I fixed both occurrences.

**Pull request merged**